peps/pep-0765.rst

```diff
@@ -0,0 +1,169 @@
+PEP: 765
+Title: Disallow return/break/continue that exit a finally block
+Author: Irit Katriel <irit@python.org>, Alyssa Coghlan <ncoghlan@gmail.com>
+Discussions-To: https://discuss.python.org/t/an-analysis-of-return-in-finally-in-the-wild/70633
+Status: Draft
+Type: Standards Track
+Created: 15-Nov-2024
+Python-Version: 3.14
+Post-History: 09-Nov-2024
```

Comment:
Let's link the date to the post:
```suggestion
Post-History: `09-Nov-2024 <https://discuss.python.org/t/an-analysis-of-return-in-finally-in-the-wild/70633>`__,
```

When this PEP is merged, please create a fresh discussion thread for the PEP, then put that link under `Discussions-To` and here under Post-History.